akka-actor-typed/src/main/scala/akka/actor/typed/delivery/internal/ConsumerControllerImpl.scala

```diff
@@ -462,7 +502,7 @@ private class ConsumerControllerImpl[A](
                 seqNr,
                 newRequestedSeqNr)
               s.producerController ! Request(confirmedSeqNr = seqNr, newRequestedSeqNr, resendLost, viaTimeout = false)
-              startRetryTimer() // reset interval since Request was just sent
+              retryTimer.start() // reset interval since Request was just sent
```

Comment:
maybe use reset here?

Reply:
The purpose here is to always reschedule since we have just sent a Request after half window size. Meaning that if there is a steady stream of incoming messages it will only send for those half window triggers and not by the scheduled interval.